frontend/packages/process-editor/src/components/Canvas/BPMNEditor/BPMNEditor.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BPMNEditor } from './BPMNEditor';
+import { BpmnApiContextProvider } from '../../../contexts/BpmnApiContext';
+import { BpmnConfigPanelFormContextProvider } from '../../../contexts/BpmnConfigPanelContext';
+import { textMock } from '../../../../../../testing/mocks/i18nMock';
+
+jest.mock('../../../hooks/useBpmnEditor', () => ({
+  useBpmnEditor: jest.fn().mockReturnValue({
+    canvasRef: { current: document.createElement('div') },
+  }),
+}));
+
+describe('BPMNEditor', () => {
+  it('render spinner when pendingApiOperations is true', () => {
+    render(
+      <BpmnApiContextProvider pendingApiOperations={true}>
+        <BpmnConfigPanelFormContextProvider>
+          <BPMNEditor />
+        </BpmnConfigPanelFormContextProvider>
+      </BpmnApiContextProvider>,
+    );
+
+    screen.getByText(textMock('process_editor.loading'));
```

Comment:
Should this test have an ```expect(...)``` to check that something is ok? 😄

Reply:
It can have, but since it is a getBy... method it would crash/fail anyway if the text is not found. So I figure it is cleaner and less boilerplate like this 🫣